src/web/routes/static/static-router.ts

```diff
@@ -2,24 +2,8 @@ import { Router, static as Static } from 'express';
 
 import { join } from 'path';
 
-import {
-	jiraAdminOnlyAuthorizationMiddleware,
-	jiraContextSymmetricJwtFromQueryAuthenticationMiddleware,
-} from '../../middleware/jira';
-
 export const staticRouter = Router();
 
-staticRouter.use(
-	'/admin/index.html',
-	jiraContextSymmetricJwtFromQueryAuthenticationMiddleware,
-	jiraAdminOnlyAuthorizationMiddleware,
-);
-
-staticRouter.use(
```

Comment:
**note:** Currently, all requests to `/static/issue-panel.html` fail with `HTTP 401`. The issue is caused by a bug in the original `jiraContextSymmetricJwtFromQueryAuthenticationMiddleware`.

<img width="547" alt="image" src="https://github.com/atlassian-labs/figma-for-jira/assets/141381034/b3d0cfe2-9206-45ad-bbaf-cb9495133e61">